src/release.py

```diff
@@ -447,6 +447,8 @@ def end_docs_build(temp_dir):
     except Exception as e:
         write_error('An exception occured in "end_docs_build": {}'.format(e))
         input("Press ENTER to continue...")
+    input('If you want to prevent "{}" to be updated, now is the good time! Press ENTER to '
```

Comment:
😉

Reply:
I need to fix the `search-index.js` generation but I think it's coming from the `minifier-rs` crate. Seems like a long time issue. :)